app/domain/mailing_list/subscribers.rb

```diff
@@ -116,4 +109,25 @@ def event_subscribers(condition)
                  Event.sti_name,
                  true)
   end
+
+  private
+
+  def excluded_people_by_tag
+    ActsAsTaggableOn::Tagging.select(:taggable_id)
```

Comment:
in my experience it's more common to use pluck instead of select.
but select is probably equal. what do you think?

Reply:
I think `.pluck` immediately fetches the records, while `.select` is used to just declare the column selection, but still keep it a pending query. So if you change to `.pluck`, it should go after the `.where`.

Reply:
Right. On line 21, I call `to_sql` to this statement. This way, theres only one sql statement being queried in the end, because this statement gets embedded into the main query.
This was already done for the `excluded_person_subscribers` and I figured this was because of performance reasons but I may be wrong about that.

Reply:
> I think `.pluck` immediately fetches the records, while `.select` is used to just declare the column selection, but still keep it a pending query. So if you change to `.pluck`, it should go after the `.where`.

of course, thx @carlobeltrame for clarifying :flashlight:

Reply:
> Right. On line 21, I call `to_sql` to this statement. This way, theres only one sql statement being queried in the end, because this statement gets embedded into the main query.
> This was already done for the `excluded_person_subscribers` and I figured this was because of performance reasons but I may be wrong about that.

then it makes sense to use `.select` here :fireworks: